cmd/jaeger/internal/integration/query_test.go

```diff
@@ -25,22 +25,17 @@ func TestJaegerQueryService(t *testing.T) {
 	t.Log("Query initialized")
 
 	// Start another instance of Jaeger receiving traces and write traces to Remote Storage
-	collector := &GRPCStorageIntegration{
-		E2EStorageIntegration: E2EStorageIntegration{
-			ConfigFile:         "../../config-remote-storage.yaml",
-			SkipStorageCleaner: true,
-			EnvVarOverrides: map[string]string{
-				// Run jaeger_query on different ports here to avoid conflict
-				// with jaeger_query instance of Jaeger above
-				"JAEGER_QUERY_GRPC_ENDPOINT": "localhost:0",
-				"JAEGER_QUERY_HTTP_ENDPOINT": "localhost:0",
-			},
+	collector := &E2EStorageIntegration{
+		ConfigFile:         "../../config-remote-storage.yaml",
+		SkipStorageCleaner: true,
+		EnvVarOverrides: map[string]string{
+			// Run jaeger_query on different ports here to avoid conflict
+			// with jaeger_query instance of Jaeger above
+			"JAEGER_QUERY_GRPC_ENDPOINT": "localhost:0",
+			"JAEGER_QUERY_HTTP_ENDPOINT": "localhost:0",
 		},
 	}
-	collector.CleanUp = collector.cleanUp
```

Comment:
Why don't we need cleanup here to purge data between tests?

Reply:
Thanks for catching! The query integration tests weren't being run in CI which was fixed in https://github.com/jaegertracing/jaeger/pull/7056. And this issue was fixed as well in this PR.